oat-sa/extension-tao-itemqti-pci
# fix/TR-2997/keep-ltr-direction-math-entry

**Related to:** [TR-2997](https://oat-sa.atlassian.net/browse/TR-2997)

**Description:**
In the Math Entry interaction, some buttons are mirrored when they should remain in the LTR direction.

**Changes:**

- Force LTR direction on LATEX symbols

**How to check:**
- if you using existing instance, you will need to run migration `sudo -u www-data php index.php '\oat\tao\scripts\tools\Migrations' -c migrate -v 'oat\qtiItemPci\migrations\Version202111240855231465_qtiItemPci' -e qtiItemPci`
- Create a PCI Math Entry item in RTL language. And duplicate it on LTR language.
- Create a test with both items and create TT with RTL language.
- Review test with the Guest user (LTR) and RTL TT.
- Check that symbols keep the appearance in the 4 scenarios (LTR + RTL / LTR + LTR / RTL + RTL / RTL + LTR)
- Check the images attached to the ticket

**TAE Environment:**

- Link to the TAE on the ticket

**Local test:**

- There is a composer link on the ticket to install it locally
